src/ibm/hypervisor-network-mgr-src/hyp_ethernet_interface.cpp

```diff
@@ -377,6 +377,37 @@ void HypEthInterface::createIPAddressObjects()
 
     auto biosTableAttrs = manager.getBIOSTableAttrs();
 
+    // The total number of vmi attributes in biosTableAttrs is 9
+    // 4 attributes of interface 0, 4 attributes of interface 1,
+    // and vmi_hostname attribute
+    if (biosTableAttrs.size() < 9)
```

Comment:
zero right?

Reply:
No, it should be 9. We cannot check if it is 0 because, this method will be called once for each ethernet interface - meaning, once for eth0 and once for eth1.
When it is called for eth0, the biosAttrs size would be 1 (as we would have already emplaced "vmi_hostname" to the biosAttrs, and we will add default values to biosAttrs with set..() method, so the next time, when this methos is called for eth1, biosAttrs size would not be 0.

Reply:
ok